samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs

```diff
@@ -34,15 +36,7 @@ public class RazorViewToStringRenderer
         public async Task<string> RenderViewToStringAsync<TModel>(string name, TModel model)
```

Comment:
We don't always require the extension `RenderViewToStringAsync("View", null)` would work just as fine. However, there are oddities with the paths we generate that prevent it from working with precompiled views. If / when we choose to fix it, we should be able to use either interchangeably.